hw/ip/keymgr/doc/_index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Key Manager HWIP Technical Specification"
+---
+
+# Overview
+
+This document specifies the functionality of the OpenTitan key manager.
+
+## Features
+
+- One-way key and identity (working) state hidden from software.
+- Version controlled identity and key generation.
+- Key generation for both software consumption and hardware sideload.
+
+
+## Description
+
+The key manager implements the hardware component of the [identities and root keys](https://github.com/lowRISC/opentitan/pull/3136) strategy of OpenTitan.
+
+It enables the system to shield critical assets from software directly and provides a simple model for software to use derived key and identity outputs.
+
+# Theory of Operation
+
+Key manager behavior can be summarized by the functional model below.
+
+![Key Manager Functional Model](keymgr_functional_model.svg)
+
+In the diagram, the red boxes represent the working state and the associated secret value, the black ovals represent derivation functions, the green squares represent software inputs, and the remaining green / purple shapes represent outputs to both software and hardware.
+
+In OpenTitan, the derivation method selected is [KMAC]().
```

Comment:
Missing link?

Reply:
ah yes, i'm waiting for Eunchan's kmac doc to officially go up, and then i'll link it in.